tee-worker/.github/workflows/publish-docker-sidechain.yml

```diff
@@ -0,0 +1,43 @@
+name: Publish Docker image for new sidechain release
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  main:
+    name: Push Integritee Sidechain to Dockerhub
+    runs-on: [ self-hosted ]
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Download sidechain from release
+        uses: dsaltares/fetch-gh-release-asset@master
+        with:
+          version: "tags/${{ github.event.release.tag_name }}"
+          file: integritee-worker-sidechain-${{ github.event.release.tag_name }}.tar.gz
+          target: "integritee-worker-sidechain.tar.gz"
+          token: ${{ secrets.GITHUB_TOKEN }}
+  
+
+      - name: Login to Dockerhub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
+
+      - name: Load Worker & Push
+        env:
+          DOCKER_BUILDKIT: 1
+        run: |
+            docker image load --input integritee-worker-sidechain.tar.gz
+            docker images --all
+            docker push integritee/sidechain:${{ github.event.release.tag_name }}
```

Comment:
Is this tag correct ?

Reply:
I think we don't check these files within `tee-worker/.github`.  They are all upstream. Not relevant to us.

Reply:
It's not used/triggered - this file is automatically created when applying the patch